.github/workflows/ci.yml

```diff
@@ -103,6 +103,7 @@ jobs:
           TELEMETRY_HEADER: ${{ secrets.TELEMETRY_HEADER }}
 
   build-image:
+    if: startsWith(github.ref, 'refs/tags/v') || github.event_name == 'workflow_dispatch'
```

Comment:
Why? It was useful to run it on PR so we make sure there is no regression, this job only takes ~1m to run.